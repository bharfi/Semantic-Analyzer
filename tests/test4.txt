scope1{
    public: a;
    a = c;
}//