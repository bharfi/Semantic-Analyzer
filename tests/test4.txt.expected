scope1.a = ?.c